source/view/wizard/pages/equations_page.py

```diff
@@ -70,7 +74,7 @@ def __init__(self, update_line_text, dictionary):
             button = QPushButton(key)
             button.setText(key)
             button.setMinimumWidth(len(key))
-            button.setFont(QtGui.QFont("Cambria Math", 9))
+            button.setFont(QtGui.QFont("DejaVu Sans", 9))
```

Comment:
```suggestion
            button.setFont(QtGui.QFont("Cambria Math", 9))
```
Needs to be changed back to Cambria Math

Reply:
I changed the font file we were using to Libertinus Math